package.json

```diff
@@ -56,7 +56,7 @@
     "ci:test:build": "yarn wsrun --serial test:build",
     "ci:test:core:components": "yarn wsrun --serial --exclude-missing ci:test",
     "ci:test:frontend": "yarn workspace blockchain-wallet-v4-frontend ci:test:frontend",
-    "clean": "cross-env yarn wsrun clean && rimraf build && rimraf coverage && rimraf dist && rimraf *.log && rimraf node_modules",
```

Comment:
when running `yarn clean` it was throwing an error (`No matches found: "*.log"`) so I removed this to make it work correctly.